src/main/java/com/ashkan/moviesapi/entities/MovieCatalog.java

```diff
@@ -1,15 +1,19 @@
-package com.ashkan.moviesapi.entity;
+package com.ashkan.moviesapi.entities;
 
 import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
 
 import javax.persistence.*;
 import java.util.Objects;
 
 @Entity
 @Table(name = "movie_catalog", schema = "moviedb", catalog = "")
 public class MovieCatalog {
```

Comment:
This Entity is a good example why is good to separate the Persistence object = Entity from the Business Model object, you added here some fields that are not needed in this Entity since this is a Persistence object.
In this case this Entity should only have the following properties:

private Movie movieByMovieId;
private Price priceByPriceId;
private Integer copies;

Just think about it :)